images/macos/templates/macOS-13.arm64.anka.pkr.hcl

```diff
@@ -34,17 +34,17 @@ variable "github_api_pat" {
   default = ""
 }
 
-variable "xcode_install_user" {
+variable "xcversion_auth_cookie" {
   type = string
-  sensitive = true
+  default = ""
 }
 
-variable "xcode_install_password" {
+variable "xcode_install_storage" {
   type = string
-  sensitive = true
+  default = ""
```

Comment:
I think we should use `sensitive = true` for both `xcode_install_storage` and `xcode_install_sas` variables to make sure that packer doesn't print these values to logs accidentally.

Reply:
done (only for hcl templates)